pkgs/by-name/mu/multipass/package.nix

```
@@ -0,0 +1,74 @@
+{
+  callPackage,
+  fetchFromGitHub,
+  lib,
+  nixosTests,
+  stdenv,
+  symlinkJoin,
+}:
+
+let
+  name = "multipass";
+  version = "1.15.0-rc4";
+
+  multipass_src = fetchFromGitHub {
+    owner = "canonical";
+    repo = "multipass";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-xwEM7Zsjkyi91oNyiX0i1mIJfBOoJ/4X47JBEqP8xYY=";
+    fetchSubmodules = true;
+    leaveDotGit = true;
+    postFetch = ''
+      # Workaround for https://github.com/NixOS/nixpkgs/issues/8567
+      cd $out
+      rm -rf .git
+    '';
```

```suggestion
    leaveDotGit = true;
    postFetch = ''
      # Workaround for https://github.com/NixOS/nixpkgs/issues/8567
      rm -rf $out/.git
    '';
```
but we do we even need leaveDotGit?